packages/liferay-npm-scripts/src/utils/validateConfig.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * Util for validating json for certain keys. If content for each key does not exist,
+ * the process will exit and emit an error with a list of keys missing. Note, this
+ * ultity does not work for keys deeper than the first level.
```

Comment:
s/ultity/utility/

May be overkill for this use case, but [JSON Schema](https://json-schema.org/) is really nice for defining what constitutes valid JSON data. I wouldn't actually add it at this point because I am allergic to dependencies, but for anything more complex than what we're dealing with here I've found it to be great for getting checks and guard clauses out of the code and into a single validation step.

Reply:
> I am allergic to dependencies
😂